routes.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+  function call($controller, $action) {
+    require_once('controllers/' . $controller . '_controller.php');
+
+    switch($controller) {
+      case 'movies':
+        $controller = new MoviesController();
+      break;
+      // case 'newController':
+      //   $controller = new newController();
+      // break;
+    }
+
+    $controller->{ $action }();
```

Comment:
Again the case against trusting raw user input here. Theoretically a user could call any method on your controller with that.

Reply:
Yes, you're right.. the pre set urls can handle this problem.